mk/def_rules.mk

```diff
@@ -262,8 +270,11 @@ ICONV := /QOpenSys/usr/bin/iconv
 ICONV_EBCDIC := IBM-037
 ICONV_ASCII := UTF-8
 SETCCSID_ASCII := 1208
-VPATH = $(OBJPATH):$(SRCPATH)
+PREUSRLIBLPATH = $(call getLibPath,$(preUsrlibl))
```

Comment:
What if there are multiple entry in the preUsrLibl or postUsrLibl
I think this code assumes that there is only 1 entry

Reply:
![Screen Shot 2022-02-03 at 9 54 24 AM](https://user-images.githubusercontent.com/17009902/152367347-0a61c74f-9af8-460b-aaad-395c96698d45.png)

Reply:
It works with multiple entries because all the text manipulate functions already works with a list.